app/actors/hyku_addons/actors/creator_institutional_email_actor.rb

```diff
@@ -18,12 +18,13 @@ def check_creator_institutional_email(env)
           return [] if creator_hash.blank?
 
           creators = creator_hash.map do |creator|
+            next if creator["creator_name_type"] == "Organizational"
```

Comment:
Don't forget to add a break return after guard statements to ensure they are easily seen by other developers reading the code